### DISCUSSION TITLE
Simplest way to delete an already indexed project from opengrok

### DISCUSSION THREAD

**Discussion by jkjha** [oracle/opengrok#4277] [General]
I am trying to figure out (basically automate later) the simplest way to remove a project from opengrok project directory.
The process that I was doing is:
Remove directory 
1. /opengrok/src/<project_name>
2.  /opengrok/data/index/<project_name>
3.  /opengrok/data/xref/<project_name>
4. /opengrok/data/suggester/<project_name>

Do I also need to delete the project info from /opengrok/etc/configuration.xml file? 

Then I came to know about the delete API.. https://opengrok.docs.apiary.io/#reference/0/project-management/delete-project
that seems the perfect way.

but then just to confirm I asked chatgpt about that... and it surprises me with response that, opengrok has admin interface which I neither know nor I can access it from my host. 

```
Q: how to delete a project from opengrok project list

A:
To delete a project from the OpenGrok project list, you need to follow these steps:

Log in to the OpenGrok server using an account that has administrative privileges.

Navigate to the OpenGrok web interface and click on the "Projects" tab.

Find the project you want to delete from the list of projects.

Click on the project to open its details page.

On the project details page, click on the "Delete" button.

A confirmation dialog box will appear. Click "OK" to confirm the deletion.

The project will be removed from the OpenGrok project list.

Note: Deleting a project from the OpenGrok project list will also delete its associated source code and indexing data. Make sure to back up any important data before proceeding with the deletion.




Q:
what is the admin UI url for opengrok?
A:
The admin UI URL for OpenGrok is typically located at http://<opengrok-hostname>:<port>/source/admin, where <opengrok-hostname> is the hostname or IP address of the OpenGrok server, and <port> is the port number on which the OpenGrok web application is running.

For example, if your OpenGrok server is running on the default HTTP port 8080 and its hostname is opengrok.example.com, the admin UI URL would be http://opengrok.example.com:8080/source/admin.

Note that the admin UI may require authentication with an administrative account in order to access certain functionality.

```

Is there really above UI exists? and if so what do I configure that?

**Comment by vladak** [comment]
That's pure nonsense, there is no admin UI. Just use the REST API to delete the project and it will take care of everything. Keep in mind that it is asynchronous API endpoint.

**Reply by jkjha** [reply]
thanks. It's surprising to see how confidently gpt is generating irrelevant responses.
